include/fam/fam.h

```diff
@@ -226,6 +245,9 @@ class Fam_Region_Descriptor {
     void set_size(uint64_t itemSize);
     void set_perm(mode_t perm);
     void set_name(char *name);
+    void set_redundancyLevel(Fam_Redundancy_Level redundancyLevel);
```

Comment:
Need to think whether we should have the "set_*" methods in the descriptor class. 
We don't want the application to use these methods to set any of the descriptor properties. These methods need to be used only by the implementation(internally).

Reply:
As discussed raised an issue to track this